src/gwt/src/org/rstudio/core/client/StringUtil.java

```diff
@@ -408,14 +408,14 @@ public static String trimBlankLines(String data)
       return data;
    }
 
-   public static String trimLeft(String str)
+   public static String trimLeft(String string)
    {
-      return str.replaceFirst("^\\s+", "");
+      return RE_LEADING_WHITESPACE.replaceAll(string, "");
```

Comment:
Small optimization -- this method was being called frequently in VirtualConsole, and pre-constructing the regular expression helps a small bit.